add patch for read-only device mappings. already accepted upstream
Index: multipath-tools/kpartx/kpartx.8
===================================================================
--- multipath-tools.orig/kpartx/kpartx.8	2013-11-05 12:41:21.399308644 +0530
+++ multipath-tools/kpartx/kpartx.8	2013-11-05 12:41:21.395308644 +0530
@@ -20,6 +20,9 @@
 .B \-r
 Read-only partition mappings
 .TP
+.B \-r
+Read-only partition mappings
+.TP
 .B \-d
 Delete partition mappings
 .TP
